webpack.analyze.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 const {targets, createConfig} = require('./webpack.common');
+// $FlowIgnore
```

Comment:
Flow has issues resolving dependencies with PnP (https://github.com/yarnpkg/berry/issues/634), but as we don't have any dependencies in main codebase, I think it's an acceptable loss.

Reply:
Since we don't have `flow-typed` for them and they don't have type information, so nothing is changed.